src/paths/properties@estimate.yaml

```
@@ -208,6 +208,15 @@ get:
       Si l'interprétabilité n'est pas jugée suffisamment fiable cet argument est ignoré."
       schema:
         type: boolean
+    - name: percentiles
+      in: query
+      description: "Liste des percentiles de la distribution du champ estimé (field=PRICE uniquement).
```

:magic_wand: chore: Ajouter apostrophes autour de `field=PRICE`.
```suggestion
      description: "Liste des percentiles de la distribution du champ estimé (`field=PRICE` uniquement).
```